tests/foreman/api/test_discoveredhost.py

```diff
@@ -136,13 +136,11 @@ def test_positive_upload_facts(self):
         """
         for name in valid_data_list():
             with self.subTest(name):
-                mac_address = gen_mac()
+                mac_address = gen_mac(multicast=False)
                 host = _create_discovered_host(name, macaddress=mac_address)
-                if bz_bug_is_open(1392919):
```

Comment:
This bug was closed as NOTABUG with comment:
> Hey, you are making POST instead GET request. This is correct behavior.
So I've added additional `read` and removed `if bz_bug_is_open` block. Though we can reopen that BZ. Cast @ldjebran